lib/ansible/modules/cloud/vmware/vmware_guest_search.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright, (c) 2018, Ansible Project
+# Copyright, (c) 2018, Moshe Immerman
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_guest_search
+short_description: Search for VM's by name
+description:
+    -  Search for VM's by name
+version_added: 2.8
+author:
+    - Moshe Immerman <moshe.immerman@gmail.com>
+notes:
+    - Tested on vSphere 6.5
+requirements:
+    - "python >= 2.6"
+    - PyVmomi
+options:
```

Comment:
Your indentation here is weird... the option names are indented with 3 spaces, then the next level is 6 spaces... please make it 4 everywhere